subsys/shell/shell.c

```diff
@@ -714,7 +717,7 @@ static int execute(const struct shell *shell)
 
 	/* Executing the deepest found handler. */
 	return exec_cmd(shell, argc - cmd_with_handler_lvl,
-			&argv[cmd_with_handler_lvl], help_entry);
+			&argv[cmd_with_handler_lvl], &help_entry);
```

Comment:
you are passing `**` here?

Reply:
no, why?

Reply:
`help_entry` is a structure, you need `&` to get structure address.

Reply:
Sorry, GitHub confused me. thought it was the same function